src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -27,6 +28,13 @@ function PIASubHeader({
   const [downloadError, setDownloadError] = useState('');
   const { pathname } = useLocation();
   const nextStepAction = pathname?.split('/').includes('nextSteps');
+  secondaryButtonText = pathname?.split('/').includes('view')
+    ? 'Edit'
+    : ' Save';
+
+  primaryButtonText = pathname?.split('/').includes('view')
+    ? 'Submit'
+    : 'Submit PIA intake';
```

Comment:
This logic if the page is a view or edit mode - should reside in the parent - PIAForm page